src/store/modules/util/actions.ts

```diff
@@ -0,0 +1,33 @@
+import { UtilService } from '@/services/UtilService'
+import { ActionTree } from 'vuex'
+import RootState from '@/store/RootState'
+import UserState from './UtilState'
+import * as types from './mutation-types'
+import { hasError } from '@/utils'
+
+const actions: ActionTree<UserState, RootState> = {
+  async fetchVarianceReasons({ commit }) {
```

Comment:
Why to use state when we use it in single page?

Reply:
Updated sir